release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -286,6 +286,75 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for vSphere. This feature was previously introduced as a Technology Preview feature in {product-title} 4.8 and is now generally available and enabled by default in {product-title} 4.10.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere[vSphere CSI Driver Operator].
+
+[id="ocp-4-10-storage-azure-disk-csi-operator"]
+==== Persistent storage using Microsoft Azure Disk CSI Driver Operator is generally available
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Azure Disk. This feature was previously introduced as a Technology Preview feature in {product-title} 4.8 and is now generally available and enabled by default in {product-title} 4.10.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure.adoc#persistent-storage-csi-azure-disk[Azure Disk CSI Driver Operator].
+
+[id="ocp-4-10-storage-aws-efs-csi-operator"]
+==== Persistent storage using AWS Elastic File Storage CSI Driver Operator is generally available
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic File Storage (EFS). This feature was previously introduced as a Technology Preview feature in {product-title} 4.9 and is now generally available and enabled by default in {product-title} 4.10.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#persistent-storage-csi-aws-efs[AWS EFS CSI Driver Operator].
+
+[id="ocp-4-10-storage-csi-migration-azure-file"]
+==== Automatic CSI migration supports Microsoft Azure file (Technology Preview)
+
+Starting with {product-title} 4.8, automatic migration for in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. This feature now supports automatic migration for the Azure File in-tree plug-in to the Azure File CSI driver.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
+
+[id="ocp-4-10-storage-csi-migration-vsphere"]
+==== Automatic CSI migration supports VMware vSphere (Technology Preview)
+
+Starting with {product-title} 4.8, automatic migration for in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. This feature now supports automatic migration for the vSphere in-tree plug-in to the vSphere CSI driver.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
```

Comment:
We should perhaps emphasize somehow that:

* This requires vSphere 6.7u3 (+ link to "CSI driver installation on vSphere clusters" above).
* This requires users to remove non-Red Hat vSphere CSI driver (+ link to https://github.com/openshift/openshift-docs/pull/40571)
* Cluster will still be upgraded if the conditions above are not met, however, users should fix the above to get a supported vSphere CSI driver.

Reply:
See my changes for this topic. Let me know if Ok.

Reply:
I am sorry, I put my comment on a wrong place. Can you please move it to "Persistent storage using VMware vSphere CSI Driver Operator is generally available"? It's not related to CSI migration that much.

Reply:
Fixed!